nano/secure/common.cpp

```diff
@@ -38,7 +38,7 @@ char const * test_genesis_data = R"%%%({
 	"source": "B0311EA55708D6A53C75CDBF88300259C6D018522FE3D4D0A242E431F9E8B6D0",
 	"representative": "xrb_3e3j5tkog48pnny9dmfzj1r16pg8t1e76dz5tmac6iq689wyjfpiij4txtdo",
 	"account": "xrb_3e3j5tkog48pnny9dmfzj1r16pg8t1e76dz5tmac6iq689wyjfpiij4txtdo",
-	"work": "9680625b39d3363d",
```

Comment:
@argakiig  previous work was almost a million times higher than test threshold, causing the difficulty to bleed into other blocks when genesis open was processed in tests